network-api/networkapi/wagtailpages/templates/wagtailpages/tags/cta/petition.html

```diff
@@ -33,7 +35,7 @@
         data-share-text="{{ cta.share_link_text }}"
         data-thank-you="{{ cta.thank_you }}"
         data-modals="{{ modals_json }}"
-	>loading CTA</div>
+	>{% trans "Loading…" %}</div>
```

Comment:
Hm, let's still mention what is actually loading?

Reply:
What about `Loading form…`? Despite the file name, this isn’t always used as an actual petition, and I think `CTA` is super cryptic

Reply:
it is, it's a pretty horrible acronym. "Loading form..." would be good.